src/typescript/sdk/src/indexer-v2/types/json-types.ts

```diff
@@ -357,6 +358,12 @@ export type DatabaseJsonType = {
       ProcessedFields &
       UserLPCoinBalance & { daily_volume: Uint128String }
   >;
+  [DatabaseRpc.PriceFeed]: Flatten<
+    Omit<MarketAndStateMetadata, "bump_time" | "market_nonce" | "trigger"> & {
+      open_price_q64: number;
+      close_price_q64: number;
```

Comment:
Maybe we can use something like what you did with `Decimal` and do`export type UnsizedNumericString = string;` for these? in `types.ts`

Would be nice so we can remember what they are